plugins/utils/autosource.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  ...
+}:
+let
+  inherit (lib.nixvim) defaultNullOpts;
+in
+lib.nixvim.vim-plugin.mkVimPlugin {
+  name = "autosource";
+  originalName = "vim-autosource";
+  package = "vim-autosource";
+  globalPrefix = "autosource_";
+
+  maintainers = [ lib.nixvim.maintainers.refaelsh ];
+
+  settingsOptions = {
+    prompt_for_new_file = defaultNullOpts.mkEnum [ 0 1 ] 1 ''
```

Comment:
You need to run `nix fmt` again, unfortunately nixfmt won't allow short lists like this to remain on a single line (yet, see https://github.com/NixOS/nixfmt/pull/233)

Reply:
Yes, I encountered this problem. I manually formatted it.

Reply:
The formatting is [still wrong](https://buildbot.nix-community.org/#/builders/1260/builds/869/steps/1/logs/stdio), running `nix fmt` really is the easiest fix :grinning:

Reply:
> Yes, I encountered this problem. I manually formatted it.

Ah re-reading, I now understand you meant that not using the auto-formatting was intentional.

To clarify, unfortunately enforcing formatting is part of our CI, and we can't merge PRs unless the CI passes.

Reply:
If you prefer, you could define a small helper in the let-block at the top of the file. This should allow the option declaration to be formatted nicer.

```nix
let
  inherit (lib.nixvim) defaultNullOpts;
  mkBoolInt = defaultNullOpts.mkEnum [
    0
    1
  ];
in
```

Then you can use:
```
foo_opt = mkBoolInt 1 ''
  Some desc
'';
```